weekly.go

```diff
@@ -247,6 +236,71 @@ func genWeeklyReportIssuesPRs(buf *bytes.Buffer, start, end string) {
 	formatSectionEndForHtmlOutput(buf)
 }
 
+func genWeeklyReportProjects(buf *bytes.Buffer, sprint *jira.Sprint) {
+	formatSectionBeginForHtmlOutput(buf)
+	buf.WriteString("\n<h1>Projects</h1>\n")
+	epicQuery := `project = %s and "Epic Link" is not EMPTY and Sprint = %d`
+	epicIssues := queryJiraIssues(fmt.Sprintf(epicQuery, config.Jira.Project, sprint.ID))
+	// An epic link set.
+	epics := make(map[string]struct{})
+	for _, is := range epicIssues {
+		// The magic name of epic link field.
+		const epicLinkField = "customfield_10100"
+		epicLink := is.Fields.Unknowns[epicLinkField].(string)
+		epics[epicLink] = struct{}{}
+	}
+
+	projects := `
+<table class="relative-table wrapped">
+  <tbody>
+  <tr>
+    <th>Name</th>
+    <th>Purpose</th>
```

Comment:
Shall we remove the Purpose to make columns less? Considering that we also have to fill descriptions, there will be too many content in one row.